### PR TITLE
Feature/a 3

### PR DESCRIPTION
**Pull request by Aberratio**

### CODE REVIEW

File: change_password.php [hunk 1]
```diff
@@ -26,108 +26,125 @@
...
+                        $get_user = "select * from bridgeplayers where email='$user'";
+                        $run_user = mysqli_query($con, $get_user);
+                        $row = mysqli_fetch_array($run_user);
+
+                        $user_pass = $row['pass'];
+                      ?>
+                    
+                      
+                      
+
```
Comment (Aberratio) [inline change_password.php:55]: unnecessary blank lines

File: lang/lang_pl.php [hunk 2]
```diff
@@ -3,7 +3,7 @@
 class Infos
 {
     // Login
-    public $bad_login_or_password = "Nieprawidłowy login lub hasło!";
+    public $bad_login_or_password = "Nieprawidłowy login lub hasło!";//invalid sounds better
```
Comment (Aberratio) [inline lang/lang_pl.php:6]: change it - remember also to modificate places that use it

File: change_password.php (hunk 1 above)
Comment (Louiks) [reply change_password.php:55]: wait, I dont have those :O

Comment (Louiks) [reply change_password.php:55]: oh i get it

Comment (Aberratio) [reply change_password.php:55]: ok, sync problems

File: lang/lang_pl.php (hunk 2 above)
Comment (Louiks) [reply lang/lang_pl.php:6]: ok

**Pull request merged**